app/views/bgeigie_imports/not_approved/index.html.haml

```diff
@@ -1,7 +1,7 @@
 = content_for(:title) { t('.bgeigie_imports') }
 
 %h2= t('.not_approved')
-
+%hr
```

Comment:
How did you end up adding these? Just noticed it while browsing the site?

Reply:
I find they make the website visually more pleasing.